apigw-lambda-rds-snapstart/example-pattern.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "Create a REST API with Amazon API Gateway, Amazon RDS and AWS Lambda with SnapStart",
+  "description": "A pattern to show how to use Lambda SnapStart with relational databases.",
+  "language": "Java",
+  "level": "300",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "CDK creates all the infrastructure we need for the pattern, include the VPC, subnets, security groups,secrets and the DB.",
+      "The DB setup Lambda function is executed once to create the database table with a given structure.",
+      "This REST API is implemented with the Micronaut framework. Credentials for the DB are passed as environment variables to the application."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/apigw-lambda-rds-snapstart",
+      "templateURL": "serverless-patterns/apigw-lambda-rds-snapstart",
+      "projectFolder": "apigw-lambda-rds-snapstart",
+      "templateFile": "do i need this?`"
```

Comment:
There are multiple CDK files, do I need this?

Reply:
This will be the code that appears in the main page of the pattern